reference-implementation/__tests__/json/packages-via-trailing-slashes.json

```diff
@@ -6,7 +6,8 @@
       "lodash-dot": "./node_modules/lodash-es/lodash.js",
       "lodash-dot/": "./node_modules/lodash-es/",
       "lodash-dotdot": "../node_modules/lodash-es/lodash.js",
-      "lodash-dotdot/": "../node_modules/lodash-es/"
+      "lodash-dotdot/": "../node_modules/lodash-es/",
+      "mapped/path/": "https://github.com/WICG/import-maps/issues/207/"
     }
```

Comment:
Also, how about adding test cases for URL-like specfiers + backtracking? (perhaps in `url-specifiers.json`, where URL-like specifiers + trailing slash is tested)